### PR TITLE
[Docs] Add Note to Readme-Template about Variable Espansion

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

File: docs/content/doc/advanced/customizing-gitea.en-us.md [hunk 1]
```diff
@@ -235,6 +235,9 @@ Locales may change between versions, so keeping track of your customized locales
 
 To add a custom Readme, add a markdown formatted file (without an `.md` extension) to `custom/options/readme`
 
+**NOTE:** readme templates support **variable expansion**.  
+currently there are `{Name}`, `{Description}`, `{CloneURL.SSH}` and `{CloneURL.HTTPS}`
```
Comment (mrsdizzie) [inline docs/content/doc/advanced/customizing-gitea.en-us.md]: Maybe a small clarification of what name this is -- is it repo name or username or full name?

Comment (bagasme) [reply docs/content/doc/advanced/customizing-gitea.en-us.md]: @mrsdizzie in the context of repo name?

Comment (6543) [reply docs/content/doc/advanced/customizing-gitea.en-us.md]: added as we discust on discord

**Review by mrsdizzie — APPROVED** [review]

**Comment by 6543** [comment]
ping lgtm

**Pull request merged**